server/item/enchantment/quick_charge.go

```diff
@@ -0,0 +1,47 @@
+package enchantment
+
+import (
+	"github.com/df-mc/dragonfly/server/item"
+	"github.com/df-mc/dragonfly/server/world"
+	"time"
+)
+
+var QuickCharge quickCharge
+
+type quickCharge struct{}
+
+// Name ...
+func (quickCharge) Name() string {
+	return "Quick Charge"
+}
+
+// MaxLevel ...
+func (quickCharge) MaxLevel() int {
+	return 3
+}
+
+// Cost ...
+func (quickCharge) Cost(level int) (int, int) {
+	minCost := 5 + (level-1)*10
+	return minCost, minCost + 20
```

Comment:
According to wiki,

```suggestion
	minCost := 12 + (level-1)*20
	return minCost, 50
```

This puts the range at 52-50 for level 3, Mojang moment :man_shrugging: